rust/connlib/shared/src/lib.rs

```diff
@@ -36,7 +36,8 @@ pub fn login_url(
     device_id: String,
 ) -> Result<(Url, StaticSecret)> {
     let private_key = StaticSecret::random_from_rng(rand::rngs::OsRng);
-    let name_suffix: String = thread_rng()
+    // FIXME: read FIREZONE_NAME from env (eg. for gateways) and use system hostname by default
```

Comment:
@conectado should I try to implement this or leave it as a TODO? I tried to make sure that it would continue to work until we started reading it properly, but not reading it creates a mess in the UI

Reply:
Would `FIREZONE_NAME` be different from `FIREZONE_ID`?

Reply:
`ID` is the unique ID that allows us to identify it when we upsert a record to a database (`external_id`) and the `name` is a string for people to identify it in the UI. 

I was thinking about merging them but the issue is that ID can be unreadable, while for `name` we want something like a `hostname`: `MacBook-Pro.local` on MacOS, `Andrew's iPhone` on iOS, etc. and we won't need to enforce its uniqueness (duplicates are bad for UX but not critical).

Reply:
So it's just an env var that **must** be present and is send as part of the URL?